tests/reals/CMakeLists.txt

```diff
@@ -43,6 +43,11 @@ set(c_util_reals_h_files
     real_uuid_string_renames.h
 )
 
+if (EXISTS ${CMAKE_CURRENT_LIST_DIR}/../../deps/smhasher/src/MurmurHash2.cpp)
+    set(c_util_reals_cpp_files ${c_util_reals_cpp_files} ${CMAKE_CURRENT_LIST_DIR}/../../deps/smhasher/src/MurmurHash2.cpp)
+    set(c_util_reals_h_files ${c_util_reals_h_files} ${CMAKE_CURRENT_LIST_DIR}/../../deps/smhasher/src/MurmurHash2.h)

```

Comment:
talking about "reals" the c_util and c_util_reals will end up exporting the same symbol from murmurhash, so inside c_utils_reals that needs to be renamed.

Parth's tool that checks that will trigger.